pkg/infra/metrics/metrics.go

```diff
@@ -195,6 +201,9 @@ var (
 
 	// MStatTotalPublicDashboards is a metric total amount of public dashboards
 	MStatTotalPublicDashboards prometheus.Gauge
+
+	// MStatTotalCorrelations is a metric total amount of correlations
+	MStatTotalCorrelations prometheus.Gauge
```

Comment:
for some reason the github merge removed this metric - https://github.com/grafana/grafana/blob/bf3ac0ff189b06ca75c5e6b241c9f70f79630f63/pkg/infra/metrics/metrics.go#L206

Reply:
took a look and it is no longer there 👍 https://github.com/grafana/grafana/blob/v9.4.x/pkg/infra/metrics/metrics.go#L111